src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -88,79 +127,109 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     }
 
     return (
-      <StructuredListWrapper border selection>
-        <StructuredListHead>
-          <StructuredListRow head>
-            <StructuredListCell head>Pipeline Run</StructuredListCell>
-            {!pipelineName && (
-              <StructuredListCell head>Pipeline</StructuredListCell>
-            )}
-            {selectedNamespace === ALL_NAMESPACES && (
-              <StructuredListCell head>Namespace</StructuredListCell>
-            )}
-            <StructuredListCell head>Status</StructuredListCell>
-            <StructuredListCell head>Last Transition Time</StructuredListCell>
-          </StructuredListRow>
-        </StructuredListHead>
-        <StructuredListBody>
-          {!pipelineRuns.length && (
-            <StructuredListRow>
-              <StructuredListCell>
-                {pipelineName ? (
-                  <span>No pipeline runs for {pipelineName}</span>
-                ) : (
-                  <span>No pipeline runs</span>
-                )}
-              </StructuredListCell>
+      <>
+        {this.state.createdPipelineRun && (
+          <InlineNotification
+            kind="success"
+            title="Successfully created PipelineRun"
+            subtitle={
+              <Link to={this.state.createdPipelineRun.url}>
+                {this.state.createdPipelineRun.name}
+              </Link>
+            }
+            hideCloseButton={false}
+            lowContrast
+          />
+        )}
+        <Button
+          iconDescription="Create PipelineRun"
+          renderIcon={Add}
+          onClick={() => this.toggleModal(true)}
+          style={{ float: 'right' }}
```

Comment:
this should be done in the stylesheet

Reply:
Ok, I added a stylesheet for the PipelineRuns container with the style.